packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingEditorState.ts

```diff
@@ -1282,6 +1310,23 @@ export class MappingEditorState extends ElementEditorState {
                 );
               }
             }
+          } else if (newMappingElement instanceof SetImplementation) {
```

Comment:
Looks like we're doing it at the wrong place here, we should move this to

```ts
// here?
throw new IllegalStateError(
                  'Expected to have current mapping editor state to be of consistent type with current mapping element',
                );
```

So we should have something like `InstanceSetImplementationEditorCompilationErrorRevealer`. Not the nicest name, I have to agree :(